src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -22,9 +24,20 @@ class ThurloeService extends Actor with MyService {
   def receive = runRoute(thurloeRoutes)
 }
 
+trait DatabaseConnectedThurloe extends ThurloeApi {
+  
+  // TODO: Fill THESE in with database access implementation.
+  def keyLookup(key: String) = Success(KeyValuePair("yek", "eulav"))
+  def collectAll() = Success(Seq(
+    KeyValuePair("key1", "Bob Loblaw's Law Blog"),
+    KeyValuePair("key2", "Blah blah blah blah blah")))
+  def setKeyValuePair(keyValuePair: KeyValuePair): Try[KeyValuePair] = Success(keyValuePair)
+  def deleteKeyValuePair(key: String): Try[Unit] = Success()
+}
+
 // this trait defines our service behavior independently from the service actor
 @Api(value="/thurloe", description = "Thurloe service", produces = "application/json", position = 1)
-trait MyService extends HttpService {
+trait ThurloeApi extends HttpService with DataAccess {
```

Comment:
should this extend `DataAccess` directly or be more like Cromwell and hold a reference to an implementation of it?